upgrade/helpers/tasks.py

```diff
@@ -95,6 +95,9 @@ def sync_capsule_repos_to_upgrade(capsules):
     RHEV_CAPSULE_AK
         The AK name used in capsule subscription
     """
+    setup_foreman_maintain()
+    logger.info('Disabling the sync plan ...')
```

Comment:
Disabling untill ``` bug ID  ``` fixes will be helpfull.

Reply:
Same has been incorporated.